kyaml/filesys/fsondisk_windows.go

```diff
@@ -5,16 +5,11 @@ package filesys
 
 import (
 	"path/filepath"
-	"testing"
 
-	"github.com/stretchr/testify/require"
 	"golang.org/x/sys/windows"
 )
 
-func getOSRoot(t *testing.T) string {
-	t.Helper()
-
+func getOSRoot() (string, error) {
 	sysDir, err := windows.GetSystemDirectory()
```

Comment:
suggest returning error early here instead of using `sysDir`:
```
if err != nil {
  return "", err
}
```

Reply:
done!